inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -150,12 +669,13 @@ std::pair<bool, std::string> compare_functions(
             return error(typeInfoToStr(type_info1) + " != " + typeInfoToStr(type_info2));
         }
 
-        auto subgraph1 = dynamic_cast<ngraph::op::util::SubGraphOp *>(node1);
-        auto subgraph2 = dynamic_cast<ngraph::op::util::SubGraphOp *>(node2);
+        auto subgraph1 = dynamic_cast<ngraph::op::util::SubGraphOp*>(node1);
+        auto subgraph2 = dynamic_cast<ngraph::op::util::SubGraphOp*>(node2);
 
         if (subgraph1 && subgraph2) {
-            auto res = compare_functions(subgraph1->get_function(), subgraph2->get_function(),
-                    compareConstValues, compareNames, compareRuntimeKeys, comparePrecisions);
+            auto res = compare_functions(
```

Comment:
Comparison of Input/Output descriptions (Slice / Merge / Invariant inputs, Concat outputs) is currently disabled? Do we have a ticket to support this?